gst/nnstreamer/tensor_query/tensor_query_server.h

```diff
@@ -40,45 +39,44 @@ typedef struct
 /**
  * @brief Get nnstreamer edge server handle.
  */
-edge_server_handle
-gst_tensor_query_server_get_handle (const char *id);
+edge_server_handle gst_tensor_query_server_get_handle (const char *id);
```

Comment:
Question)
The form of API signature is changed. Is there any special reason?
* Before
```cpp
edge_server_handle
gst_tensor_query_server_get_handle (const char *id);
```
* After
```cpp
edge_server_handle gst_tensor_query_server_get_handle (const char *id);
````

Reply:
The API is not changed and was automatically changed by the gst-indent tool.